Packs/IntegrationsAndIncidentsHealthCheck/ReleaseNotes/1_2_0.md

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
please follow https://confluence.paloaltonetworks.com/display/DemistoContent/Release+Notes+And+Descriptions